src/contributor-guide/dod/tco-analysis.md

```diff
@@ -0,0 +1,105 @@
+---
+group: dod
+title: DoD - TCO Analysis
+---
+
+## Overview
+
+All functional changes must be analysed for the impact on merchant total cost of ownership (TCO) and the results of the analysis should be added to the pull request or issue description to meet the "Definition of Done" (DoD).
+
+The goal of TCO analysis is to identify:
+
+-  If the changes have impact on TCO
+-  The value of the impact, that consists of
+   -  The level of the change (patch/minor/major)
+   -  Potential number of affected extensions/customisations (several/many/most/all)
+
+### Changes that have impact on TCO
+
+Changes can impact total cost of ownership if the extensions or customisations, established processes or integrations within the projects build on Adobe Commerce framework may need to be adjusted to accommodate the changes.
+
+Changes that have impact on TCO:
```

Comment:
may be we need to say "an example of changes affecting TCO"
This list is not exhaustive

Reply:
I don't think we can call it an example, but yes, it is also not an exhaustive list (it does not say so). It is just a list of "Changes that have an impact on TCO". Cannot think about other proper name variants that are not confusing

Reply:
maybe "Some changes that have an impact on TCO"

Reply:
Changed